sources/packages/web/src/composables/useFormatters.ts

```diff
@@ -304,6 +309,21 @@ export function useFormatters() {
     return `${negativeValueDisplay}$${Math.abs(currencyValue).toFixed(2)}`;
   };
 
+  /**
+   * Validates if the given date 1 is before the given date 2.
+   * @param date1 given date 1.
+   * @param date2 given date 2.
+   * @returns if the given date 1 is before the given date 2.
+   */
+  const isBeforeDate = (
+    date1: string | Date,
+    date2: string | Date,
+  ): boolean => {
+    return dayjs(dayjs(date1).format("YYYY-MM-DD")).isBefore(
```

Comment:
`dayjs` has a second parameter that allows providing the expected format, something like:
`dayjs(date1, "YYYY-MM-DD")`

Reply:
@andrewsignori-aot the 2nd parameter is dayjs(), is actually for the validation check of the 1st parameter. For the format, `.format()` is to be used
https://day.js.org/docs/en/parse/is-valid#docsNav
https://day.js.org/docs/en/parse/string-format

Reply:
As discussed, updated comment and updated the method name